src/js/ConfirmAlert.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export default class ConfirmAlert extends React.Component {
+    constructor() {
+        super();
+    }
+
+    render() {
+        return (
+            <div className="alert">
+                <div className="alert-title">
+                    <p className="t-small t-light th-f-color">
+                        {this.props.name}
+                    </p>
+                </div>
+                <div className="alert-top">
+                    <h4 className="t-small t-medium th-f-color">{this.props.description}</h4>
+                    <img src={this.props.iconUrl} />
+                </div>
+                <div className="alert-buttons">
+                    <div className="alert-button-2 th-f-color t-small t-light th-bg-color th-soft-buttons" 
+                        key={0} onClick={this.props.leftCallback}>
+                            {this.props.leftText}
+                    </div>
+                    <div className="alert-button-2 th-f-color t-small t-light th-bg-color th-soft-buttons" 
+                        key={1} onClick={this.props.rightCallback}>
+                            {this.props.rightText}
+                    </div>
+                </div>
+            </div>
+        );
+    }
+}
```

Comment:
new line please

Reply:
added in https://github.com/smartdevicelink/generic_hmi/pull/240/commits/9420e8cd33100df3cba2420ebea12091814adcac